demo/index.html

```diff
@@ -1,80 +1,77 @@
 <!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
 <html>
   <head>
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
 
-    <title>Performance Benchmark Demo</title>
+    <title>Test Benchmark Demo</title>
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-    <link rel="import" href="../performance-benchmark.html">
+    <link rel="import" href="../test-benchmark.html">
+    <link rel="import" href="../../paper-styles/demo-pages.html">
+    <style is="custom-style">
+      body {
+        height: 100vh;
```

Comment:
is this `fullbleed`? there's an `iron-flex-layout` mixin for that (i think you might need to nuke the margin too, which that does)